modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Options/Options.es.js

```diff
@@ -322,9 +318,7 @@ const Main = ({
 					<KeyValue
 						generateKeyword={option.generateKeyword}
 						keyword={option.value}
-						keywordReadOnly={
-							defaultLanguageId !== editingLanguageId
-						}
+						keywordReadOnly={false}
```

Comment:
@carolmariaabb, what is the default value of the property `keywordReadOnly`? If it's `false`, then we can remove this line.

Reply:
@natocesarrego This attribute has no default value. Because of this, I kept this line.

Reply:
Amazing!